LVT.Tests/LVTParseTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using LVT.LVT.Services;
+using NUnit.Framework;
+using System.IO;
+namespace LVT.Tests
+{
+    [TestFixture]
+    public class ParseJSONLVTFromStreamTest
+    {
+        [Test]
+        public void ParseJsonLVT()
+        {   //this opens the TestLVT file stored in the project's resources
+            Stream TestStream = new MemoryStream(Properties.Resources.TestLVT);
+            StreamReader TestStreamReader = new StreamReader(TestStream);
+
+            JsonParser TestParser = new JsonParser();
+            LeanValueTree TestSingleBranchLVT = TestParser.ParseJsonLVTFromStream(TestStreamReader);
+
+            Assert.IsInstanceOf(typeof(LeanValueTree), TestSingleBranchLVT, "ParseJsonLVT should return LeanValueTree object");
+            Assert.AreNotEqual(null, TestSingleBranchLVT.Vision, "ParseJsonLVT should populate objects inside the LVT");
+        }
+    }
+
+    [TestFixture]
+    public class LVTObjectTests
+    {
+        LeanValueTree SingleBranchLVT;
+
+        [SetUp]
+        public void Setup()
+        {
+            Stream stream = new MemoryStream(Properties.Resources.TestLVT);
+            StreamReader streamReader = new StreamReader(stream);
+            JsonParser Parser = new JsonParser();
+            SingleBranchLVT = Parser.ParseJsonLVTFromStream(streamReader);
+        }
+
+         [Test]
+        public void SingleBranchLVT_Vision_Tests()
+        {
+            Vision VisionObj = SingleBranchLVT.Vision;
+
+            Assert.AreEqual("visionTitle", VisionObj.Title, "Vision should have title");
+            Assert.IsInstanceOf(typeof(Vision), VisionObj, "Test Vision should be instance of Vision Class");
+            Assert.AreEqual(1, VisionObj.Goals.Count, "Test Vision should have one goal");
```

Comment:
Should this also be tested with multiple goals/branches with another JSON?

Reply:
Currently we are just checking singleBranch LVT.  Yes, we can check it for multiple goals/branches as well.